.github/workflows/get-environment.yml

```
@@ -577,6 +581,16 @@ jobs:
             core.setOutput("releaseScope", releaseScope);
             core.setOutput("isCloud", isCloud);
 
+      - name: "Detect delivery type"
+        id: detect_delivery_type
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          script: |
+            if (${{ contains(github.event.pull_request.labels.*.name, 'database') && 'true' || 'false' }}) { const delivery_type = 'feature' }
+            else { delivery_type = 'classic' }
```

this will fail because `delivery_type` is not declared in `else`
```suggestion
            let delivery_type  = 'classic';
            if (${{ contains(github.event.pull_request.labels.*.name, 'database') && 'true' || 'false' }}) {
              delivery_type = 'feature';
            }
```